R/bundle.R

```diff
@@ -425,18 +358,15 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters,
     verbose = verbose
   )
 
-  pyInfo <- NULL
-  needsPyInfo <- appUsesPython(quartoInfo) || "reticulate" %in% names(packages)
-  if (needsPyInfo && !is.null(python)) {
-    pyInfo <- inferPythonEnv(appDir, python, condaMode, forceGenerate)
-    if (is.null(pyInfo$error)) {
-      # write the package list into requirements.txt/environment.yml file in the bundle dir
-      packageFile <- file.path(appDir, pyInfo$package_manager$package_file)
-      cat(pyInfo$package_manager$contents, file = packageFile, sep = "\n")
-      pyInfo$package_manager$contents <- NULL
-    } else {
-      stop(paste("Error detecting python environment:", pyInfo$error), call. = FALSE)
-    }
+  needsPython <- appUsesPython(quartoInfo) || "reticulate" %in% names(packages)
```

Comment:
If this is the only use, maybe pass `packages` into `appUsesPython`?

Reply:
Or maybe inline `appUsesPython` here?

Reply:
Either option feels fine, and not a permanent decision should we want to further refactor.